src/App.tsx

```diff
@@ -1,204 +1,62 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+import { Errors } from './types/Errors';
+import { Footer } from './components/Footer';
+import { ErrorModal } from './components/ErrorModal';
+import { FilterBy } from './types/FilterBy';
+
+const filter = (todos: Todo[], filterBy: FilterBy) => {
+  switch (filterBy) {
+    case FilterBy.All:
+      return todos;
+    case FilterBy.Active:
+      return todos.filter(todo => !todo.completed);
+    case FilterBy.Completed:
+      return todos.filter(todo => todo.completed);
+    default:
+      return todos;
```

Comment:
```suggestion
  switch (filterBy) {
    case FilterBy.Active:
      return todos.filter(todo => !todo.completed);
    case FilterBy.Completed:
      return todos.filter(todo => todo.completed);
    case FilterBy.All:
    default:
      return todos;
```
you can combine cases